src/plugins/imagery/mixins/imageryData.js

```diff
@@ -54,7 +54,7 @@ export default {
         this.timeKey = this.timeSystem.key;
         this.timeFormatter = this.getFormatter(this.timeKey);
 
-        this.telemetryCollection = this.openmct.telemetry.requestCollection(this.domainObject, {});
+        this.telemetryCollection = this.openmct.telemetry.requestCollection(this.domainObject, {}, this.timeContext);
```

Comment:
Is there a reason we aren't attaching the `timeContext` to the `options` object?

Reply:
Let me give that a go. It's been a minute, but I remember [this ticket](https://github.com/nasa/openmct/issues/6493) at least erupted from looking at this.

Reply:
@ozyx I moved the `standardizeRequestOptions` up so we can could reliably have a `timeContext` to listen to, but otherwise it seems to behave the same. Good idea!